commit 787e20e5de4fb0d95f6d5094cf829d248406f1f4 (refs/changes/03/3803/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2018-04-11T11:09:03-07:00 (1 year, 6 months ago)
    
    OS-6887 test-vm "test log archiving" failure: new console.log and platform.log surprised test case

diff --git a/src/vm/tests/test-docker.js b/src/vm/tests/test-docker.js
index 8f2af6dd..d6fe538b 100644
--- a/src/vm/tests/test-docker.js
+++ b/src/vm/tests/test-docker.js
@@ -1080,8 +1080,10 @@ test('test log archiving', function (t) {
         }, function (cb) {
             var log_dir = '/zones/' + state.uuid + '/logs';
 
-            // ensure we have 2 log files now
+            // Ensure we have 2 "stdio.log*" files now.
             fs.readdir(log_dir, function (err, files) {
+                files = files.filter(
+                    function (f) { return f.indexOf('stdio.log') === 0; });
                 t.ok(files.length === 2, 'vm logs: ' + JSON.stringify(files));
                 cb();
             });
